Gopkg.toml

```diff
@@ -34,3 +34,7 @@ required = [
 [[constraint]]
   name = "cloud.google.com/go"
   version = "0.17.0"
+
+[[constraint]]
+  name = "github.com/google/go-cmp"
```

Comment:
I think @kargakis's point about using the api machinery instead probably still stands?